Documentation/ReleaseNotes.md

```diff
@@ -90,6 +90,7 @@ In some cases there was a tradeoff that had to be made:
 See [Leap Motion Hand Tracking Configuration](CrossPlatform/LeapMotionMRTK.md) for the extra integration step.
 For those who are using ARFoundation, there's now an additional manual step in its getting started steps.
 See [ARFoundation](CrossPlatform/UsingARFoundation.md#install-required-packages) for the new steps.
+For those who will be using [Holographic Remoting](Tools/HolographicRemoting.md) on HoloLens 2, there is now a manual step to perform.
```

Comment:
Nit: Add a link to the #HoloLens-2 section for HolographicRemoting.md

Reply:
fixed in next commit